<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YandexModePanel</class>
 <widget class="QWidget" name="YandexModePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Яндекс Диск</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Токен:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leToken">
       <property name="text">
        <string>AQAAAABIWyynAAccdsvhLvBIeEIHom_UA_a9Rog</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btnGetToken">
       <property name="text">
        <string>Получить токен</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QLineEdit" name="leFilePath"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Файл:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Загрузить файл:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QPushButton" name="btnUploadFile">
       <property name="text">
        <string>Отправить на Яндекс Диск</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnBrowseFile">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lePassword">
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Зашифрованные файлы на Яндекс Диск:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Удалить файл</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Обновить список</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QTableView" name="tvFileList"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnDownload">
       <property name="text">
        <string>Скачать файл</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="leProgress">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
